cypress/integration/Basic Flows/component_library.spec.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
+
+context('Component Library', () => {
+  before(()=> {
+    cy.loginWithUI()
+
+    //make sure that there is a project to work with when only running this file
```

Comment:
It will be necessary when testing in high environments.  We will either need to add some seed data or create it.  I want to test the component library by using the dropdown to test the adding a component to my project but the wrong projects were available in the dropdown due to the refactor work that you are doing...